src/css/graph.css

```diff
@@ -90,12 +90,27 @@ svg.scatter-plot {
   background-color: #00ed89;
 }
 
+.key-circle.red::before {
+  background-color: #CD5958;
+}
+
 .key-circle.green-with-opacity::before {
```

Comment:
Should we get rid of the unused CSS?

Reply:
Or alternatively use black and white for these rather than the color it displays as. Since that's how we do it in other parts of the code

Reply:
Done